test/versioning/semver-cargo.spec.js

```diff
@@ -0,0 +1,68 @@
+const semver = require('../../lib/versioning/semver-cargo');
+
+describe('semver.matches()', () => {
+  it('handles comma', () => {
+    expect(semver.matches('4.2.0', '2.0, >= 3.0 < 5.0.0')).toBe(true);
+    expect(semver.matches('4.2.0', '4.0.0, 3.0.0')).toBe(true);
+    expect(semver.matches('4.2.0', '4.3.0, 3.0.0')).toBe(false);
+    expect(semver.matches('4.2.0', '4.3.0, 3.0.0, > 5.0.0 <= 6.0.0')).toBe(
+      false
+    );
+  });
+});
+describe('semver.maxSatisfyingVersion()', () => {
+  it('handles comma', () => {
+    expect(
+      semver.maxSatisfyingVersion(
+        ['0.4.0', '0.5.0', '4.0.0', '4.2.0', '5.0.0'],
```

Comment:
What would happen if you added `4.2.1` to this array? I've never seen this type of range pattern before and doesn't make much sense to me.